sviluppo/review.rst

```diff
@@ -26,29 +27,33 @@ Scelta della PR
 
 La lista di tutte le PR aperte è https://github.com/OCA/l10n-italy/pulls.
 
-Le PR pronte per essere verificate sono identificate dalla label **needs review**.
+Le PR pronte per essere verificate sono identificate dall'etichetta **needs review**.
```

Comment:
Questo non è più vero :) Si tratta di un'etichetta che usiamo quasi mai!

Reply:
> Questo non è più vero :) Si tratta di un'etichetta che usiamo quasi mai!

In effetti era vero quando ho aperto la PR lo scorso anno... :sweat_smile: 

Ho cambiato l'immagine e il testo, prova a vedere se può andare.

Reply:
Perfetto, grazie 1K!